src/sql/parts/notebook/notebook.css

```diff
@@ -23,4 +23,8 @@
 	text-align: center;
 	display: flex;
 	align-items: center;
+}
+
+.notebookEditor .actionbar-container .monaco-action-bar > ul.actions-container {
```

Comment:
Is this for horizontal toolbar? You got rid of actionbar-container from the vertical one.

Reply:
Yeah, it's for the horizontal toolbar, but let me double check what it looks like in the vertical one.